#use standard header
Header header

uint8 device         # index of the device to control
# 0 - Head
# 1 - Base

uint8 r              #red intensity percentage (0-100)
uint8 g              #green intensity percentage
uint8 b              #blue intensity percentage